### ISSUE TITLE
Admin tab js breaks other pages.

### ISSUE THREAD

**Issue by matt-h** [wp-stream/stream#708]
We have a plugin which adds some standard admin tabs to the user-edit.php page. Since ui/js/admin.js runs on this page it is running tab script on that page with the following selector which breaks the page since the tabs are the same one Stream is looking for.
```
var $tabs          = $( '.nav-tab-wrapper' ),
```

Should scope the selector to make sure it is grabbing only your tabs instead of all WordPress Admin tabs.

**Comment by lukecarbis** [comment]
Thanks @matt-h . That's a good idea. Would you mind creating a PR for it?

**Comment by matt-h** [comment]
There wasn't any unique stream elements around the settings page to update the selector to so one would need to be added. I've also no idea where else in stream this code might be being used.

I just wrapped the `// Admin page tabs` section with `if(window.location.href.indexOf("wp_stream_settings") !== -1)` which was a quick fix so that code only ran on the settings page.

I saw that you had `$script_screens = array( 'plugins.php', 'user-edit.php', 'user-new.php', 'profile.php' );` but I'm not sure why the admin js was running on the user pages.
